File: go/0412_fizzbuzz.go
package main

import "strconv"

func fizzBuzz(n int) []string {
	res, val := make([]string, n), ""
	for i := 1; i <= n; i++ {
		switch {
		case i%15 == 0:
			val = "FizzBuzz"
		case i%3 == 0:
			val = "Fizz"
		case i%5 == 0:
			val = "Buzz"
		default:
			val = strconv.Itoa(i)
		}
		res[i-1] = val
	}
	return res
}

File: python/0412_fizzbuzz.py
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        return [self.convert(x) for x in range(1, n + 1)]

    def convert(self, num):
        if num % 3 == 0 and num % 5 == 0:
            return 'FizzBuzz'
        elif num % 3 == 0:
            return 'Fizz'
        elif num % 5 == 0:
            return 'Buzz'
        return str(num)